script/deploy-dev.mk

```diff
@@ -5,32 +5,36 @@
 # that has been setup with OIDC support (see ./cluster-kind.mk)
 
 deploy-dex: devel/dex.crt devel/dex.key
-	kubectl create namespace dex
-	kubectl -n dex create secret tls dex-web-server-tls \
+	kubectl --kubeconfig=${CLUSTER_CONFIG} create namespace dex
+	kubectl --kubeconfig=${CLUSTER_CONFIG} -n dex create secret tls dex-web-server-tls \
 		--key ./devel/dex.key \
 		--cert ./devel/dex.crt
-	helm install dex stable/dex --namespace dex --version 2.13.0 \
-		--values ./docs/user/manifests/kubeapps-local-dev-dex-values.yaml
+	helm --kubeconfig=${CLUSTER_CONFIG} repo add stable https://charts.helm.sh/stable
+	helm --kubeconfig=${CLUSTER_CONFIG} install dex stable/dex --namespace dex  --values ./docs/user/manifests/kubeapps-local-dev-dex-values.yaml
 
 deploy-openldap:
-	kubectl create namespace ldap
-	helm install ldap stable/openldap --namespace ldap \
+	kubectl --kubeconfig=${CLUSTER_CONFIG} create namespace ldap
+	helm --kubeconfig=${CLUSTER_CONFIG} repo add stable https://charts.helm.sh/stable
+	helm --kubeconfig=${CLUSTER_CONFIG} install ldap stable/openldap --namespace ldap \
 		--values ./docs/user/manifests/kubeapps-local-dev-openldap-values.yaml
 
+# Get mkcert from https://github.com/FiloSottile/mkcert/releases
 devel/localhost-cert.pem:
-	mkcert -key-file ./devel/localhost-key.pem -cert-file ./devel/localhost-cert.pem localhost 172.18.0.2
+	mkcert -key-file ./devel/localhost-key.pem -cert-file ./devel/localhost-cert.pem localhost 172.19.0.2
```

Comment:
Oh - good point, we need to include `deploy-dev.mk` in the files requiring updating (to get a cert for the correct IP - I was wondering why my browser still warned me yesterday when I tried the setup).

Reply:
Ups, this file was uploaded with my config (172.19.x.x, instead of 172.18.x.x), but yes, I should include it in that list of files.

Reply:
Ups, this was an unintended change, I just uploaded my config. However, as you mention, yep, we should add this file into that list